shop/src/pages/Main.js

```diff
@@ -82,13 +104,39 @@ const Main = () => {
   const { config } = useConfig()
   const isMobile = useIsMobile()
   const [menu, setMenu] = useState(false)
+  const [policies, setPolicies] = useState([['', '', false]])
+  useEffect(() => {
+    fetch(`shop/policies`, {
+      method: 'GET',
+      headers: {
+        authorization: `bearer ${encodeURIComponent(config.backendAuthToken)}`
+      }
+    })
+      .then((res) => {
+        if (!res.ok) {
+          return null
+        } else {
+          return res.json() //Expected type: <Array<Array<String, String, boolean>>>
+        }
+      })
+      .then((result) => {
+        if (result) {
+          setPolicies(result)
+        }
+      })
+      .catch((err) => {
+        console.error('Failed to load shop policies', err)
```

Comment:
Should display an error in the UI

Reply:
This fetch happens when a DShop customer lands on the main (public-facing) page of the shop. In the scenario where an error occurs during the fetch, I just expect that the footer (of the main page) display links to everything but the shops policies -- for example, **About** and **Contact**. If you still think an error should be displayed in the UI, could you please share your reasoning?

Reply:
Got it. Yes, that's reasonable. An error causing the footer to not display is probably not worthwhile surfacing to the user. Thanks for the explanation.